src/test/java/com/ibm/cloud/sdk/core/test/BaseServiceUnitTest.java

```diff
@@ -80,4 +91,44 @@ protected static MockResponse jsonResponse(Object body) {
   protected static MockResponse errorResponse(int statusCode) {
     return new MockResponse().setResponseCode(statusCode);
   }
+
```

Comment:
Added a couple of utility methods to this test base class to be used by the authenticator tests.